texar/modules/embedders/embedder_base.py

```diff
@@ -142,3 +148,12 @@ def forward(self,  # type: ignore
         mask += input_tensor.new_empty(noise_shape).uniform_(0, 1)
         mask = torch.floor(mask).div_(keep_rate)
         return input_tensor * mask
+
+    @property
+    def output_size(self) -> int:
+        r"""The output size, here output size equals to the input tensor size.
+
+        :returns:
+            - 1, since the output dimensions are equals to input dimensions.
```

Comment:
No need to use a list when we have only one bullet point. This is also confusing because it could be read as `-1`.